src/box/mp_error.cc

```diff
@@ -169,12 +169,6 @@ mp_sizeof_error_one(const struct error *error)
 	return data_size;
 }
 
```

Comment:
The tests do not pass on the branch - you didn't update `.result` file. Also please add `NO_DOC` and `NO_CHANGELOG` labels to your commit.

Reply:
done